### PR TITLE
algo/patch2: optins cannot have arguments

### PR DESCRIPTION
**Pull request by jumpsiegel**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1460)

### CODE REVIEW

File: algorand/wormhole_core.py [hunk 1]
```diff
@@ -460,6 +461,9 @@ def verifyVAA():
                                     # What signatures did this verifySigs check?
                                     s.store(Gtxn[i.load()].application_args[1]),
 
+                                    # Make sure we bale earlier on incorrect arguments...
```
Comment (SEJeff) [inline algorand/wormhole_core.py:464]:
```suggestion
                                    # Make sure we bail earlier on incorrect arguments...
```

**Review by SEJeff — DISMISSED** [review]

File: algorand/wormhole_core.py (hunk 1 above)
Comment (jumpsiegel) [reply algorand/wormhole_core.py]: resolved

**Review by tbjump — APPROVED** [review]
lgtm

**Review by SEJeff — APPROVED** [review]

**Review by panoel — DISMISSED** [review]

**Review by SEJeff — APPROVED** [review]

**Review by tbjump — APPROVED** [review]
lgtm

**Pull request merged**